pkg/postgres/schema/report_configurations.go

```diff
@@ -43,5 +43,6 @@ type ReportConfigurations struct {
 	Id         string                                 `gorm:"column:id;type:varchar;primaryKey"`
 	Name       string                                 `gorm:"column:name;type:varchar"`
 	Type       storage.ReportConfiguration_ReportType `gorm:"column:type;type:integer"`
+	ScopeId    string                                 `gorm:"column:scopeid;type:varchar"`
```

Comment:
This does not seem right, need to check.

Reply:
n_52_to_n_53
m_171_to_m_172
If I upgrade to 3.74 and then upgrade to 4.0, this field will not be filled. I know it is not in our original plan because we said we do not upgrade production cluster.

Reply:
`m_178_to_m_179` should be the migration to populate this.